<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EveTradeMaster</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="newPredictionsTab">
         <attribute name="title">
          <string>New predictions</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="inputIdsLe">
              <property name="placeholderText">
               <string>4, 6, 1337</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newPredictionBtn">
              <property name="text">
               <string>Prediction from line</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="defaultPredictionBtn">
              <property name="text">
               <string>Prediction from default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="unsavedPredictionsTw">
            <property name="textElideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>13</number>
            </property>
            <row/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="savedPredictionsTab">
         <attribute name="title">
          <string>Saved predictions</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="savedPredictsFilterLabel">
                <property name="text">
                 <string>Sort by: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="savedPredictionsFilterCmbb">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>FIlter</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="savedPredictionsTw">
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="columnCount">
               <number>13</number>
              </property>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="typesInDBTab">
         <attribute name="title">
          <string>Types in db</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="savedItemsFilterLabel">
              <property name="text">
               <string>Sort by: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="typesFIlterCmbb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addTypesBtn">
              <property name="text">
               <string>Add more types</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="savedTypesTw"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
